tests/cxx/overload_expr-i4.h

```diff
@@ -7,7 +7,16 @@
 //
 //===----------------------------------------------------------------------===//
 
+#ifndef INCLUDE_WHAT_YOU_USE_TESTS_CXX_OVERLOAD_EXPR_I4_H_
+#define INCLUDE_WHAT_YOU_USE_TESTS_CXX_OVERLOAD_EXPR_I4_H_
+
 // All the OverloadedFn2 overloads visible without ADL are in the same file.
 
 void OverloadedFn2(int);
 void OverloadedFn2(char);
+
+// Variable template.
+template <typename>
+int var_tpl;
```

Comment:
I was a bit surprised not to see:
```
template <typename T>
T var_tpl;
```
here, but I guess the result is the same?

Reply:
Yep, I tried that locally, and no change. OK, then I prefer the `int` version, because it's the simplest.